### PR TITLE
Pull request deployments

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary
This PR adds the ability to deploy a pull heroku review app for pull requests.

Additional clean up is required, but this is a good first start.

Thanks for contributing to Kaiju.
@cerner/kaiju

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1 +1,2 @@
 **/node_modules
+**/.git
```
Comment (ryanthemanuel) [inline .dockerignore]: Why is this necessary out of curiosity?

Comment (mjhenkes) [reply .dockerignore]: I could just do .git. There is no need to copy the git files to the docker container. The intent is to make for a smaller container.

Comment (mjhenkes) [reply .dockerignore]: The "**/" will ignore the .git folder in any subdirectory.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,62 @@
+FROM redis:3.2 AS redis
+
+FROM traefik:1.4.4 AS traefik
+
+FROM node:8.4.0 AS node
+
+FROM ruby:2.4.2
```
Comment (StephenEsser) [inline Dockerfile]: This is different than our .ruby-version file, should be bump that file too?

Comment (mjhenkes) [reply Dockerfile]: yes, but not with this PR. I want to bump the node version as well.
